app.js

```diff
@@ -0,0 +1,123 @@
+const express = require('express');
+const morgan = require('morgan');
+
+const app = express();
+
+app.use(express.json());
+app.use(morgan('dev'));
+
+app.get('/users', async (req, res) => {
+    try {
+        const pageNumber = req.query.page;
+        const response = await fetch(`https://reqres.in/api/users?page=${pageNumber}`);
+        const users = await response.json();
+
+        if (!users) {
+            res.status(404).json({ message: 'Users Not found' });
+        }
+
+        res.status(200).json(users);
+    } catch (err) {
+        res.status(500).json({ message: 'Server Side Error' });
+    }
+});
+
+app.get('/users/:id', async (req, res) => {
+    try {
+        const id = req.params.id;
+        const url = `https://reqres.in/api/users/${id}`;
+        const response = await fetch(url);
```

Comment:
send all fetch calls to another file, name it `api.js`, that contains 4 methods `get | post | put | delete`